worlds/stardew_valley/__init__.py

```diff
@@ -126,7 +122,7 @@ def create_items(self):
 
         locations_count = len([location
                                for location in self.multiworld.get_locations(self.player)
-                               if not location.event])
+                               if location.address is not None])
```

Comment:
Wouldn't this be simpler? Or maybe it's not the same and I missed something?
```suggestion
                               if location.address])
```

Reply:
Address of 0 is technically not an event. It is reserved though, so it's not like your suggestion would ever fail. 

`is not None` could technically be faster, because it only needs to compare two pointers, but currently it isn't in py3.11, i think.

Reply:
is  not None  is the kind of thing I'd expect to be fast in Cython though.

Reply:
So I was missing something. Thanks!

Reply:
I mean, it should be further faster by just dropping the []

Reply:
I don't think Generator are Sized? Buy indeed something like `sum(1 for location in locations if location.address is not None)` would probably be faster